.buildkite/scripts/build_pr.sh

```diff
@@ -1,2 +1,97 @@
 #!/bin/bash
-echo "nothing to see yet"
+set -euo pipefail
+set +x
+
+# This script should only be invoked by the Buildkite PR bot
+if [ -z ${GITHUB_PR_BRANCH+set} ] || [ -z ${GITHUB_PR_TARGET_BRANCH+set} ] || [ -z ${GITHUB_PR_NUMBER+set} ] || [ -z ${GITHUB_PR_BASE_REPO+set} ];then
+  echo "One of the following env. variable GITHUB_PR_BRANCH, GITHUB_PR_TARGET_BRANCH, GITHUB_PR_NUMBER, GITHUB_PR_BASE_REPO is missing - exiting."
+  exit 1
+fi
+
+# Configure the git author and committer information
+export GIT_AUTHOR_NAME='Buildkite CI'
+export GIT_AUTHOR_EMAIL='docs-status+buildkite@elastic.co'
+export GIT_COMMITTER_NAME=$GIT_AUTHOR_NAME
+export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
+
+# Set some metadata for build filtering capabilities
+# https://buildkite.com/elastic/docs-build-pr/builds?meta_data[repo_pr]=tech-content_123
+buildkite-agent meta-data set "repo_pr" "${GITHUB_PR_BASE_REPO}_${GITHUB_PR_NUMBER}"
```

Comment:
Would it be helpful to set separate metadata for the repo name and the PR number (either instead of, or in addition to, this field) ? I can see that as being useful for checking "all the builds running on this repo", not just "all the builds that have run on this PR in this repo".